rust/kernel/print.rs

```diff
@@ -439,3 +748,43 @@ macro_rules! pr_cont (
         $crate::print_macro!($crate::print::format_strings::CONT, true, $($arg)*)
     )
 );
+
+/// Printk ratelimit control with a shared [`ratelimit_state`]. The state will be
+/// shared with all this macro & kernel's [`printk_ratelimit`] callers.
+///
+/// Equivalent to the kernel's [`printk_ratelimit`] function.
+///
+/// Returns `true` if ratelimit control is enforced, otherwise `false`.
+///
+/// [`where`] parameter is a string literal that will be used as an
+/// identifier when ratelimite is triggered. In C this is populated with
+/// __function__.
+///
+/// # Examples
+///
+/// ```
+/// # use kernel::prelude::*;
+/// # use kernel::pr_cont;
+///
```

Comment:
Empty line (I guess the generated docs will show it), note that the `#` ones are discarded.